markdown/blog/2025-Q1-marketing-report.md

```diff
@@ -0,0 +1,190 @@
+|  Tweets      | 89         |      162   |-44%  |
+| Retweets        | 24       |       54     |  -53% |
+|Impressions       |         14,621|    27,823  |  -46% |
+|Engagements     |       278  |   543      |  -48% |
+|Replies     |      3  |  2     | +75% |
+| Clicks       |    101       |     199      |  -48% |
+|Likes          |       93   |        188      |  -49% |
+|New Followers|         36     | -27| +239%|
+
+### Key Insights
+
+Tweet output dipped by 44%, leading to a -46% dip in impressions as well.
+Replies saw a 75% increase, but overall reply numbers remain low.
+New followers count rose by +239%, given the dip and lose of followers in the previous quarter. 
+
+Overall, the decline in some metrics can be attributed to the higher number of events in the previous quarter, which naturally led to more content and activity compared to this quieter quarter.
+
+### Website Performance 
+
+<Figure
+  src="/img/posts/marketing-images/goggle-analysis-page-view.webp"
+   alt="A table showing AsyncAPI  Google Analytics Homepage performance from January to March"
+  caption="AsyncAPI Google Analytics Homepage performance from January to March"
+  className="text-center"
+/>
+
+
+**User Engagement Insights (Google Analytics)**
+**Homepage Performance**
+
+The main landing page, "AsyncAPI Initiative for event-driven APIs," received the highest traffic (17.3% of total views) but saw relatively low average time on page (17s), suggesting users may be skimming or using it as a jumping-off point.
+
+**Strong Interest in Version 3.0.0**
+
+The "3.0.0" documentation page held 16.6% of the total views with a solid 41s average time, showing deep interest in the newest spec release.
+
+**High Engagement Pages**
+
+"AsyncAPI document structure" and "Payload schema" had fewer visits but strong time-on-page (44s and 36s respectively), indicating users are spending time understanding these technical sections.
+
+Similarly, "Understanding AsyncAPIs with a Practical Example" drew good engagement (28s avg.) and "Coming from OpenAPI" shows decent dwell time (30s), making them great learning pathway pages.
+
+**Notable Drops**
+
+"Generator" and "Tools" saw good traffic but experienced drops in average time spent (-13.4% and +1.9%, respectively). This might mean users don’t find these pages as intuitive or relevant and could benefit from clearer CTAs or improved content structure.
+
+## Event & Community Engagement
+
+### Google Summer of Code (GSoC) Mentorship Program
+
+- AsyncAPI has successfully joined Google Summer of Code 2025, providing mentorship opportunities to aspiring open-source contributors.
+- [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator Elegbede (Ace) Azeez, to guide potential mentees.
+- Outcome: Increased awareness and participation in the AsyncAPI open-source ecosystem.
+
+### Community Spotlight Initiative
```

Comment:
It would be best to add the link to the blog

Reply:
Which blog?